Client.xcodeproj/project.pbxproj

```diff
@@ -7081,7 +7081,7 @@
 					"-lxml2",
 					"$(inherited)",
 				);
-				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_RELEASE -DMOZ_TARGET_CLIENT -DPAID";
+				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_RELEASE -DMOZ_TARGET_CLIENT -DPAID -DBETA";
```

Comment:
@naira-cliqz  you don't need to add this as it is being added by fastlane anyway

Reply:
Are you sure? :)

Reply:
yeah, in Fastfile you will find the following:
https://github.com/ghostery/browser-ios/blob/master/fastlane/Fastfile#L275